src/main/java/nl/novi/techiteasydto/mappers/TelevisionDtoMapper.java

```diff
@@ -0,0 +1,29 @@
+package nl.novi.techiteasydto.mappers;
+
+import nl.novi.techiteasydto.models.ResponseTelevisionDto;
+import nl.novi.techiteasydto.models.Television;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Hoewel het feitelijk niet veel uit maakt, is het om semantische redenen beter om hier de generieke `@`Component annotatie te gebruiken, omdat `@`Service impliceert dat het om een service laag gaat. Dat is hier niet het geval. Functioneel zijn beide annotaties hetzelfde.